### PR TITLE
docs: update/clean up the beyond bitswap project

### PR DESCRIPTION
**Pull request by daviddias**

### CODE REVIEW

File: BEYOND_BITSWAP/README.md [hunk 1]
```diff
@@ -15,49 +15,57 @@ In high level, this project is about:
...
 
+## 💌 Invite to Research with us
+
+ResNetLab collabores with over 10 Research Groups all over the world and Protocol Labs Research has developed research collaborations in multiples of ten in the last few years. We are always eager to collaborate with more researchers in all kinds of capacity, from thesis project (M.Sc or PhD), to Post-Doc, Grants, RFPs and independent research projects.
+
+We are making all our contributions, ideias, testbed, benchmarking and analysis scripts below. You are more than welcome to pick on these and drive one, or build on top. If you have questions, please [mail us](mailto:resnetlab@protocol.ai).
+
 ## Contributions & Results
 
 ### Documents
```
Comment (daviddias) [inline BEYOND_BITSWAP/README.md:31]: @adlrocha can you make sure that the docs shared have the following perms:

image

File: BEYOND_BITSWAP/README.md [hunk 2]
```diff
@@ -15,49 +15,57 @@ In high level, this project is about:
...
+
 * [Related Work](https://docs.google.com/document/d/14AE8OJvSpkhguq2k1Gfc9h0JvorvLgOUSVrj3CnOkQk/edit#heading=h.nxkc23tlbqhl): It gives an overview of the problem, how it will be tackled, and a collection of references and community proposals.
-* [Beyond Bitswap Slides](https://docs.google.com/presentation/d/18_aRTye2t6Xs_VhKwEbhvCYYu9ePaLgamIrJkpUDtfY/edit#slide=id.p): Set of slides introducing the project and summarizing the Related Work document from above. These slides were used to introduce the project in the following [talk]().
+* [Beyond Bitswap Slides](https://docs.google.com/presentation/d/18_aRTye2t6Xs_VhKwEbhvCYYu9ePaLgamIrJkpUDtfY/edit#slide=id.p): Set of slides introducing the project and summarizing the Related Work document from above. These slides were used to introduce the project in the following [talk](???).
```
Comment (daviddias) [inline BEYOND_BITSWAP/README.md:34]: @adlrocha this is missing a url

File: BEYOND_BITSWAP/README.md [hunk 3]
```diff
@@ -15,49 +15,57 @@ In high level, this project is about:
...
+* [Testbed, benchmarking, analysis scripts and related assets](https://github.com/protocol/beyond-bitswap/): All the code used for the implementation and other auxiliary testing assets. Additional documentation is provided in the repo.
 * [Bitswap fork](https://github.com/adlrocha/go-bitswap): This fork of `go-bitswap` is the one being used to implement and evaluate some of the RFCs and where additional metrics that want to be tracked in the testbed are being included. RFCs are imeplemented in different branches with the code of the RFC.
-* [Graphsync fork](): This fork of `go-graphsync` is also being used to test some of the RFCs.
-* [New exchange protocol?](): This is a work in progress to be determined with the result of our explorations.
 
 ### Talks
-* [Introduction to Beyond Bitswap project](): Introductory talk to show the initial work and motivate the project.
+
+* [Introduction to Beyond Bitswap project](??): Introductory talk to show the initial work and motivate the project.
```
Comment (daviddias) [inline BEYOND_BITSWAP/README.md:62]: @adlrocha this is missing a url

File: BEYOND_BITSWAP/README.md (hunk 2 above)
Comment (adlrocha) [reply BEYOND_BITSWAP/README.md:34]: We may have to remove this link because it was from the first talk that I gave which, unfortunately, no one recorded.

**Pull request merged**